entrypoint.sh

```diff
@@ -18,7 +18,6 @@ function feedback() {
   case "${1}" in
     ERROR)
       >&2 echo -e "${!color}${1}:  ${2}${DEFAULT}"
-      exit 1
```

Comment:
This exit was causing the goat to exit on the first error found. I removed it in order to handle exiting an reporting failures downstream.